code/game/machinery/adv_med.dm

```diff
@@ -15,6 +15,7 @@
 		/obj/item/weapon/stock_parts/scanning_module
 	)
 	var/mob/living/carbon/occupant
+	var/obj/item/device/antibody_scanner/immune
```

Comment:
This looks a bit silly to me
Don't forget to set it to null/qdel'ing it in `Destroy()`

Reply:
I qdel/null'd the ref in destroy.